spec-main/window-helpers.ts

```diff
@@ -7,18 +7,13 @@ export const closeWindow = async (
   { assertNotWindows } = { assertNotWindows: true }
 ) => {
   if (window && !window.isDestroyed()) {
-    const isClosed = emittedOnce(window, 'closed')
-    window.setClosable(true)
-    window.close()
-    await isClosed
+    window.destroy()
```

Comment:
`window.destroy()` will not be a synchronous signal if the window embeds a `<webivew>`. It is still better to wait for close signal before returning function control.

Reply:
The thing that's causing flake here is that in some cases, especially when the test itself closes the window, waiting for the 'closed' event doesn't work because the window is already closed. the `isDestroyed` check doesn't seem to be a good enough signal to determine whether or not a `closed` event will be emitted.

Reply:
In that case the tests that are closing windows can use `.destroy()` api , so that `isDestroyed` will become a reliable signal ?

Reply:
I'm worried about future flakes. This behaviour is far from obvious to test writers / reviewers. The fix needs to be resilient to future flakes.